src/index.js

```diff
@@ -10,8 +14,18 @@ module.exports = function (sequelize) {
       const comp = authorization.split(' ');
       if (comp.length == 2 && comp[0] == 'Bearer') {
         const token = comp[1];
+        /* Before we can assign the user id, we need to make sure the token
+         * is properly signed. This will mitigate the improper authentication
+         * attempts.
+         */
+        jwt.verify(token, secret, function(err, data) {
+          if (err)
+          {
+            response.status(401).json({ error: "Invalid Token" });
+            throw new Error("Invalid Token");
```

Comment:
Do we catch the Error somewhere?
As I know, it could crash the app if the error isn't properly caught.

Reply:
Correct. At the moment I was thinking, since this is a middleware, the endpoint can handle the exception. That may have been a bad idea. 

I have now changed it return a 401 response. Hope that is correct.